### PR TITLE
Brigade 2020 Annual Review

### PR DESCRIPTION
**Pull request by vdice**
Adds the 2020 [Brigade](https://github.com/brigadecore/brigade) Annual Review per the [CNCF Sandbox project Annual Review process](https://github.com/cncf/toc/blob/master/process/sandbox-annual-review.md).

Readable link can be seen [here](https://github.com/vdice/cncf-toc/blob/docs/2020-brigade-annual-review/reviews/2020-brigade-annual.md).

Thank you!

### CODE REVIEW

**Comment by technosophos** [comment]
What is the process with this PR going forward? Is it to be merged? We (the Brigade maintainers) are not sure what else we are supposed to do.

**Comment by technosophos** [comment]
Just checking in on this. We are unclear about the current policies, and are wondering if we need to go ping individual TOC members for a review, or if this entire process is no longer necessary with the new sandbox layout.

**Comment by michelleN** [comment]
On today's TOC call, @amye asked for TOC sponsors to review this project so that is what we're waiting on. Thanks for your submission. Nothing more is required on your end. We'll work to better document expectations around response time and further clarify the process (tracked in #489 and #490)

File: reviews/2020-brigade-annual.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- *What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?*
+  - Make significant strides towards Brigade 2.0
+  - Improve notes and awareness for Brigade releases
+  - Increase rate of new contributors
+  - We want to set up Brigade to move into incubation in 2022.
+
+- *How can the CNCF help you achieve your upcoming goals?*
+    - We believe opportunities like occasional CNCF webinars or blog posts are good for informing the CNCF user base of our project's features and goals.
+    - When the 2.0 planning cycle ends and development begins, we feel that we need to develop a platform for reaching new contributors.
```
Comment (alena1108) [inline reviews/2020-brigade-annual.md:81]: Is there an approximate ETA of 2.0 planning cycle completion, and the development kickoff?

**Comment by alena1108** [comment]
Things that I've found good about the project:

* Good docs and developer guide.
* Well documented releases
* Thought out proposal for Brigade 2.0 https://github.com/krancour/brignext/blob/master/PROPOSAL.md with the detailed and frank overview of early design decisions.
* An honest reflection on Incubation readiness:

```
While we believe we have made progress toward that goal, we want to release Brigade 2.0 before entering incubation.
```

I have left one comment on the proposal about estimated date for Brigade 2.0 development kickoff. Otherwise, LGTM on Sandbox annual review.

File: reviews/2020-brigade-annual.md (hunk 1 above)
Comment (vdice) [reply reviews/2020-brigade-annual.md]: Anticipated timeline added in-line to this section of the document.  Thank you!

**Comment by krancour** [comment]
The 2.0 proposal was made public just now. This is an up to date link: https://github.com/brigadecore/brigade/pull/1101

This PR may need to be amended accordingly.

**Comment by lizrice** [comment]
I'm happy to support Sandbox annual review

**Comment by bridgetkromhout** [comment]
Hi, @amye! I see that we have 2 of the required 3 sponsors, and according to the [public working doc](https://docs.google.com/document/d/1jpoKT12jf2jTf-2EJSAl4iTdA7Aoj_uiI19qIaECNFc/edit) we also have a week before the meeting where "Annual Review for Sandbox moves to simple majority vote". What action would you recommend at this point if we wanted to wrap up Brigade's annual review under the terms we started it with? Thanks!

**Comment by amye** [comment]
This is designed to be much easier for the projects under annual review, moving to a simple majority vote will happen in a closed TOC meeting and will be much faster! 
However, one more TOC sponsor will close this out before August 11th's meeting.

**Comment by michelleN** [comment]
LGTM on sandbox annual review from me.

**Comment by bridgetkromhout** [comment]
Hi, @amye! I see you've moved this to a "Complete" column - can this PR be merged? Or should it be closed? Thanks.

**Comment by amye** [comment]
Knew someone was missing!

**Pull request merged**